pkg/events/event_store.go

```diff
@@ -80,6 +81,14 @@ func (es *EventStore) CollectEvents() []*si.EventRecord {
 	return messages
 }
 
+// test only
+func (es *EventStore) CollectInternalEvents(idx uint64) []*si.EventRecord {
```

Comment:
As it is used by testing only, we should use lowercase letter to expose it to same package only.

Reply:
Removed the test only method.